docs/DeveloperGuide.md

```diff
@@ -1,34 +1,283 @@
 # Developer Guide
 
```

Comment:
Perhaps you can add a table of contents and employ clickable(s) to make the DG more navigable.

Reply:
You can also add in an introduction to your Development Guide that includes purpose of documentation, intended reader, brief description of your product.